feat: minor fixes and doc updated (#2193)
SchwarzIT/onyx: apps/docs/src/development/index.md

## Installation

::: tip Prerequisites
We assume that you already have a Vue application set up.

> If thats not the case we recommend you first hop over to the [Vue Quickstart Docs](https://vuejs.org/guide/quick-start.html#creating-a-vue-application).
> Follow the steps there, then you will have a Vue application and are ready to continue here.

Reviewer: ```suggestion
If thats not the case we recommend you first hop over to the [Vue Quickstart Docs](https://vuejs.org/guide/quick-start.html#creating-a-vue-application), follow the steps there and then continue with the onyx setup here.
```
